TLDR.md

```diff
@@ -0,0 +1,15 @@
+# TLDR for the TLDR
+
+We're building a secure, decentralized communication protocol. Protocol v1.0.0 Jasper has multiple REST API servers in a mesh network. Clients connect to different TX (transmission) and RX (reception) servers. Servers can detect and report misbehaving servers. Everything is signed by users' keys.
```

Comment:
@Luke-Oldenburg can you review here plz

Reply:
No server misbehavior detection yet. TX and RX servers are identical. Clients will only hit specific endpoints from each one they choose.